src/wp-includes/class-wp-classic-to-block-menu-converter.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * WP_Classic_To_Block_Menu_Converter class
+ *
+ * @package WordPress
+ * @since 6.3.0
+ */
+
+/**
+ * Converts a Classic Menu to Block Menu blocks.
+ *
+ * @since 6.3.0.
+ * @access public
+ */
+class WP_Classic_To_Block_Menu_Converter {
+
+	/**
+	 * Converts a Classic Menu to blocks.
+	 *
+	 * @since 6.3.0.
+	 * @param WP_Term $menu The Menu term object of the menu to convert.
```

Comment:
We generally separate `since` and `params` with an empty line (see [example](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods))